src/extension/healthStatusIndicator.ts

```diff
@@ -0,0 +1,49 @@
+import {
+    ExtensionContext,
+    StatusBarAlignment,
+    StatusBarItem,
+    window,
+} from "vscode";
+
+import { pluginName } from "./coqPilot";
+
+export class HealthStatusIndicator {
```

Comment:
Yep, for sure it's not the `Health` status: I feel it usually means that something can break down. Maybe our class should be more like `PluginStatusIndicator` or something like that